### ISSUE TITLE
Strange timezone string in the system information

### ISSUE THREAD

**Issue by kmizumar** [cloudera-ps/prereq-checks#59]
`Timezone:      ->`

Not sure from which commit, but fdb64051ae4f1d5d25a05dbdbcbf65d3d2989a94 shows the above on both of CentOS7 and CentOS6.

**Comment by kmizumar** [comment]
Found the root cause of this. When the symbolic link to `/etc/localtime` is created on a date less than 10, two consecutive space characters will be shown between the month and the day. 

```
[root@maru-centos69 prereq-checks]# ls -l /etc/localtime 
lrwxrwxrwx 1 root root 39 Jul  7 01:04 /etc/localtime -> /usr/share/zoneinfo/America/Los_Angeles
[root@maru-centos69 prereq-checks]# ls -lh /etc/localtime 
lrwxrwxrwx 1 root root 39 Jul  7 01:04 /etc/localtime -> /usr/share/zoneinfo/America/Los_Angeles
```

In the above example, "Jul" "[space]" "[space]" "7" are displayed and then the field number is changed from 11 to 12.
```
  local timezone=`ls -lh /etc/localtime | cut -d' ' -f11 | cut -d'/' -f5-`
```

**Comment by kmizumar** [comment]
This problem is addressed by #61 Fix strange timezone string in system information.

**Comment by kmizumar** [comment]
#61 is merged.
